BUILD

```diff
@@ -181,6 +181,13 @@ cc_library(
     deps = [":protobuf_lite"],
 )
 
+cc_library(
+    name = "protobuf_headers",
+    hdrs = glob(["src/**/*.h"]),
+    includes = ["src/"],
+    visibility = ["//visibility:public"],
```

Comment:
Please restrict the visibility to tensorflow only and add a TODO pointing to the missing feature in bazel.